openquake/qa_tests_data/disagg/case_8/job.ini

```diff
@@ -0,0 +1,54 @@
+[general]
+
+description = Disaggregation test
+calculation_mode = disaggregation
+random_seed = 23
+
+[geometry]
+
+sites = 0.0 0.0
+
+[logic_tree]
+
+number_of_logic_tree_samples = 0
+
+[erf]
+
+rupture_mesh_spacing = 2.0
+width_of_mfd_bin = 0.1
+area_source_discretization = 5.0
+
+[site_params]
+
+reference_vs30_type = measured
+reference_vs30_value = 760.0
+reference_depth_to_2pt5km_per_sec = 1.0
+reference_depth_to_1pt0km_per_sec = 50.0
+
+[calculation]
+
+source_model_logic_tree_file = sscLt.xml
+gsim_logic_tree_file = gmcLt.xml
+investigation_time = 1.0
+intensity_measure_types_and_levels = {
+  "SA(0.1)": logscale(0.005, 2.13, 45),
+  }
+truncation_level = 6
+maximum_distance = 200.0
+
+[disaggregation]
+
+poes_disagg = 0.002105, 0.000404
+mag_bin_width = 0.2
+distance_bin_width = 25.0
+coordinate_bin_width = 100.0
+num_epsilon_bins = 8
```

Comment:
Is this needed? I would have expected num_epsilon_bins = 1.

Reply:
Yes it's needed because in this case the probability of exceedance is all assigned to one epsilon bin (but we still want to know which one contributes the most ultimately)